package.json

```diff
@@ -16,11 +16,11 @@
     "prettier": "prettier --write \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "prettier-check": "prettier --check \"./**/*.{md,jsx,json,html,css,js,yml}\"",
     "test": "pnpm jest",
-    "jest": "cross-env NODE_ENV=test LOG_LEVEL=error MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --",
+    "jest": "cross-env NODE_ENV=test LOG_LEVEL=error FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --",
```

Comment:
Nice, I forgot about these.

You could go a bit further here.  These variables should get defined in the configs, like you're doing with the mocks.   The only place we really need these now is in `test/jest.config.js` and `test/jest.setup.js`.

In another PR, can you do the same sort of thing that you're doing with search, and set these variables on `process.env` in the `test/jest.setup.js` file?  Then we can remove all of them from here, and maybe get rid of `cross-env` too?

Reply:
Yup, I'll look into doing that in another PR